build-system/tasks/storybook/index.js

```diff
@@ -85,7 +84,7 @@ function buildEnv(env) {
     [
       './node_modules/.bin/build-storybook',
       `--config-dir ${configDir}`,
-      `--output-dir ${repoDir}/examples/storybook/${env}`,
+      `--output-dir ${process.cwd()}/${dir}/${env}`,
```

Comment:
Is it always safe to assume that cwd would be the root directory? Or do we even care? // cc: @rsimha

Reply:
All `amp` tasks start at the repo root before loading the task code even if they are invoked from an inner directory, so this is safe.

https://github.com/ampproject/amphtml/blob/ae3ccd197a6bd3d569f4e582a1560ca561c4fff1/build-system/task-runner/amp-task-runner.js#L67-L77